src/mono/wasm/Wasm.Build.Tests/BuildTestBase.cs

```diff
@@ -834,6 +836,33 @@ protected void AssertBlazorBundle(string config, bool isPublish, bool dotnetWasm
                         dotnetJsPath!,
                         "Expected dotnet.native.js to be same as the runtime pack",
                         same: dotnetWasmFromRuntimePack);
+
+            string bootConfigPath = Path.Combine(binFrameworkDir, "blazor.boot.json");
+            Assert.True(File.Exists(bootConfigPath), $"Missing '{bootConfigPath}'");
```

Comment:
nit: Instead of `Missing ..` use `Expected to find ..` to follow the existing pattern.